EdvClient.js

```diff
@@ -585,7 +585,7 @@ export class EdvClient {
     if(count === true) {
       return response.data;
     }
-    const docs = response.data;
+    const docs = response.data.documents;
```

Comment:
it might be better to destructure here:

```js
const {documents} = response.data;
```

I think we were assigning data to docs before so there was an obvious semantic meaning which having `documents` in the response.data now covers.

Reply:
If you're going to destructure: :)
```
const {data: {documents}} = response;
```